### DISCUSSION TITLE
Can public methods be available as variables in the view like blade components?

### DISCUSSION THREAD

**Discussion by felixdorn** [livewire/livewire#2494] [1. Help] [answered]
Blade Component already implements this behaviour and afaik it would be trivial to reproduce it.

I can definitely submit a PR but I'm not sure if I missed something or if this is an intended non-feature.

Please let me know if this is already possible,
Félix

**Comment by ju5t** [comment]
There are [computed properties](https://laravel-livewire.com/docs/2.x/properties#computed-properties). I am not sure what you're after as you didn't include an example, but I think this comes close.

**Reply by felixdorn** [reply]
Thanks for your time and answer, appreciate it!

**Comment by joshhanley** [comment, marked as answer]
@felixdorn there was a PR https://github.com/livewire/livewire/pull/2352 submitted for that, but it caused breaking changes, so had to be reverted. Have a look at the discussion on the PR.

Hope this helps!

**Reply by felixdorn** [reply]
Thanks for your time and answer, appreciate it!
